vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh

```diff
@@ -101,7 +101,7 @@ installBpftrace() {
     local bpftrace_url="https://upstreamartifacts.azureedge.net/$bpftrace_bin/$version"
     local bpftrace_filepath="/usr/local/bin/$bpftrace_bin"
     local tools_filepath="/usr/local/share/$bpftrace_bin"
-    if [ "$(isARM64)" = 1 ]; then
```

Comment:
Did we have a bug here?!!  Shouldn't it be == ? lol

Reply:
The old way was also fine. There are different ways to do a comparison. Some work some don't and it's easily getting confused. For this case specifically, 
- using `=` wasn't the best practice but still worked. Since 1 is numerical value, the best practice is using `-eq`. 
  - `=` is comparing strings and it will automatically convert the values to strings, even it's an integer. So for this case, even it was `=`, they would be treated as strings to compare.
- `==` is not POSIX compliant (means it's specific to BASH only thus less portable). But in terms of logic, it will work.